### PR TITLE
Solve some conflicts on readme

### PR DESCRIPTION
**Pull request by danielmelogpi**
Tries to solve issue #5 
Waiting revisions

### CODE REVIEW

**Comment by danielmelogpi** [comment]
Cause I'm lazy, here is the file on the last commit: 
https://github.com/danielmelogpi/nylas-mail/blob/01002152df275fd33f7b857061acfa0ee4d1b424/README.md

**Comment by lxalln** [comment]
This looks good to me. Nicely put.

**Comment by danielmelogpi** [comment]
@philip-ulrich , fixed it

**Comment by mayurdw** [comment]
1. Add a link in the download section for the previous versions of the project which the company deployed. 
2. Line 12, we should not say Alas when talking about the name of the project
3. For now, remove the Build Status icon since we haven't figured out our builds yet

I think everything else is awesome

**Comment by philip-ulrich** [comment]
LGTM. 

I think if you hit the points @mayurdw, this will be a good start.

**Comment by dweremeichik** [comment]
@mayurdw @danielmelogpi  don't remove the build badge, point it to: https://travis-ci.org/nylas-mail-lives/nylas-mail

**Comment by danielmelogpi** [comment]
the commit 67ab3f5d5c07a2b6ec4874f8fbc1ba19761a9041 addresses @mayurdw's points

**Comment by danielmelogpi** [comment]
Solved a conflict and I believe we are good to merge this one

File: README.md [hunk 1]
```diff
@@ -114,7 +126,9 @@ When you install packages, they're moved to ~/.nylas-mail/packages, and Nylas Ma
 You can configure Nylas Mail in a few ways—for instance, pointing it to your self-hosted instance of the sync engine or changing the interface zoom level. [Learn more about how.](https://github.com/nylas/nylas-mail/blob/master/CONFIGURATION.md)
 
 ## Feature Requests / Plugin Ideas
-Have an idea for a package or a feature you'd love to see in Nylas Mail? Search for existing [GitHub issues](https://github.com/nylas/nylas-mail/issues) and join the conversation!
+
+Have an idea for a package or a feature you'd love to see in Nylas Mail? Search for existing [GitHub issues](https://github.com/nylas-mail-lives/nylas-mail/issues) and join the conversation!
```
Comment (dweremeichik) [inline README.md]: Url can be shortened to /issues

Comment (danielmelogpi) [inline README.md]: `[GitHub issues](/issues)` results in a broken link.

**Comment by dweremeichik** [comment]
@danielmelogpi sorry for the relative link issues. I could have sworn I've used it and it worked before. Is this good to go in your eyes?

**Comment by mayurdw** [comment]
I think we are ready to merge this one

**Comment by dweremeichik** [comment]
Is that the new slack link?

**Comment by danielmelogpi** [comment]
@dweremeichik , no problem at all :)
I believe we are ready to go too, @mayurdw 
I'd like to have one of you to accept the request, just out of ritual procedure

**Comment by dweremeichik** [comment]
@mayurdw please make sure to squash, and follow previous commit message styles.

**Pull request merged**